opengrok-indexer/src/test/java/org/opengrok/indexer/analysis/PlainAnalyzerTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ */
+package org.opengrok.indexer.analysis;
+
+import org.apache.lucene.document.Document;
+import org.junit.jupiter.api.Test;
+import org.opengrok.indexer.analysis.plain.PlainAnalyzer;
+import org.opengrok.indexer.analysis.plain.PlainAnalyzerFactory;
+import org.opengrok.indexer.configuration.RuntimeEnvironment;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStreamWriter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class PlainAnalyzerTest {
+
+    private static StreamSource getStreamSource(final byte[] bytes) {
+        return new StreamSource() {
+            @Override
+            public InputStream getStream() throws IOException {
+                return new ByteArrayInputStream(bytes);
+            }
+        };
+    }
+
+    @Test
+    void testXrefTimeout() {
+        RuntimeEnvironment env = RuntimeEnvironment.getInstance();
+        long timeoutOriginal = env.getXrefTimeout();
+        int timeout = 3;
```

Comment:
Is not `1` enough? Saving 2 seconds for each build can be good.

Reply:
good point, fixed.